exporters/otlp/otlp_exporter_benchmark.cc

```diff
@@ -0,0 +1,122 @@
+#include "otlp_exporter.h"
+
+#include <benchmark/benchmark.h>
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace exporter
+{
+namespace otlp
+{
+
+const int kBatchSize = 200;
+
+const trace::TraceId kTraceId(std::array<const uint8_t, trace::TraceId::kSize>(
+    {0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1}));
+const trace::SpanId kSpanId(std::array<const uint8_t, trace::SpanId::kSize>(
+    {0, 0, 0, 0, 0, 0, 0, 2}));
+const trace::SpanId kParentSpanId(std::array<const uint8_t, trace::SpanId::kSize>(
+    {0, 0, 0, 0, 0, 0, 0, 3}));
+
+// OtlpExporterTestPeer is a friend class of OtlpExporter
+class OtlpExporterTestPeer
+{
+public:
+  std::unique_ptr<sdk::trace::SpanExporter> GetExporter(
+      std::unique_ptr<proto::collector::trace::v1::TraceService::StubInterface> &stub_interface)
+  {
+    return std::unique_ptr<sdk::trace::SpanExporter>(
+        new exporter::otlp::OtlpExporter(std::move(stub_interface)));
+  }
+};
+
+class FakeServiceStub : public proto::collector::trace::v1::TraceService::StubInterface
+{
+  grpc::Status Export(grpc::ClientContext *context,
+                      const proto::collector::trace::v1::ExportTraceServiceRequest &request,
+                      proto::collector::trace::v1::ExportTraceServiceResponse *response) override
+  {
```

Comment:
Nothing critical, but some compilers throw a warning which says "unused parameter" if a parameter is not used inside the function body.

I think void-ing out the parameters in this function (and the ones below) like `void <param>` might help achieve compiler portability?

Just curious to know everybody's thoughts on this.

Reply:
Seems reasonable to me, thank you! I've updated the code to void the parameters

Reply:
Please resort to [unnamed parameters](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#Rf-unused). That's the recommended way to avoid those warnings.

Reply:
I see. That makes sense. Thank you!